core/src/main/scala/org/apache/spark/deploy/rest/StandaloneRestClient.scala

```diff
@@ -241,7 +241,7 @@ private[deploy] class StandaloneRestClient extends Logging {
       }
     } else {
       val failMessage = Option(submitResponse.message).map { ": " + _ }.getOrElse("")
-      logError("Application submission failed" + failMessage)
+      logError(s"Application submission failed$failMessage")
```

Comment:
it looks like its missing a space between the failed and message but then again it wasn't here before.  What does this message look like?